### DISCUSSION TITLE
BOT_ACCESS_TOKEN

### DISCUSSION THREAD

**Discussion by raghurama123** [lucydot/ChooChoo#50] [Q&A]
I will probably try this tool in the future.

**Comment by lucydot** [comment]
Hi @raghurama123  - great! And apologies I have only just seen your message....

I'll email you the access token. Let me know if you have any issues setting up choochoo bot. It feels a little complicated to be honest, so I may have to work on streamlining the process. Happy to help if you come across any problems.

If you do use it, I think you will be the first person (beyond myself) to do so. I'll be using it for teaching this autumn term coming.

**Comment by raghurama123** [comment]
Thank you, @lucydot. I am planning to automate the exam/grading component of my [Numerical Methods course](https://github.com/raghurama123/NumericalMethods). I came across ChooChoo in June when you sent me the JOSS article for review.  

I have now forked the choochoo-template and following the steps in the documentation. Thanks for sending the token. I have included this in a test repository [raghurama123/NumMethods](https://github.com/raghurama123/NumMethods).
 
Something seems to have changed in github

    Go to Settings -> Pages.
    Select the drop-down menu below Source.
    Select gh-pages.

I do not see 'gh-pages' as an option. Instead, I see 'GitHub Actions' and 'Deploy from a branch'. How does this change affect the ChooChoo configuration?

**Comment by lucydot** [comment]
@raghurama123 thanks for highlighting the change - I've updated the docs: https://lucydot.github.io/ChooChoo/instructors/setup/ .

I've also added another step - you need to add `choochoo-bot` as a collaborator to your repo  (Settings -> Collaborators -> Add people) - let me know once you do this, and I'll accept the invitation.
